DOC-696: Add new `image_file_types` setting to paste and powerpaste (#1744)
tinymce/tinymce-docs: _includes/configuration/image-file-types.md

### `image_file_types`

> This option allows you to specify which image file formats will be recognised and modified by [smart_paste](#smart_paste) when pasted.

Reviewer: ... recognised and modified by smart_paste
would "processed" be suitable in this context?

Reviewer 2: `you` is normally a banned word because it's personifying the text or something to that effect. Can this be reworded? eg:
```suggestion
This option configures which image file formats will be recognised and modified by the [smart_paste](#smart_paste) functionality when content is pasted into the editor.
```

> This option configures which image file formats will be recognised and modified by the [smart_paste](#smart_paste) functionality when content is pasted into the editor.

Reviewer 3: ```suggestion
This option configures which image file formats will be recognized and modified by the [`smart_paste`](#smart_paste) functionality when content is pasted into the editor.
```
US spelling. 
Also, please mention what it's doing with these file formats?

Author: The `smart_paste` docs explain what happens, should I reiterate it here?

Reviewer 3: a little. It's not clear that this option determines which converts urls to images... all content needs to be able to stand on it's own to some extent, as we can't control how people will enter the docs, read the docs, or how much they will read.

> This option configures which image file formats will be recognized and placed in an `img` element by the [smart_paste](#smart_paste) functionality when content is pasted into the editor.

Reviewer 3: ```suggestion
This option configures which image file formats will be recognized and placed in an `img` element by the [`smart_paste`](#smart_paste) functionality when content is pasted into the editor.
```

**Type:** `String`

**Default Value:** `'jpeg,jpg,jpe,jfi,jfif,png,gif,bmp,webp'`

> **Possible Values:** A list of valid web image file formats. For a list of possible values see: [Image file type and format guide](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types).

Reviewer 2: Should this mention "extensions"? as that's what this setting accepts.

Reviewer 3: some people call them extensions, so please add this.

Reviewer 3: Also, prefix the link text with `MDN Web Docs - `
important to make clearer that they are leaving our site for this info.

#### Example: Using `image_file_types`

```js
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: '{{plugincode}}',

>   smart_paste: true,

Reviewer 2: Smart paste is enabled by default, so should that be included here?

Author: I thought it was important to have it here as even though it's default, it's required for this functionality. I can add a comment saying it's defaulted to `true`?

Reviewer 3: Please add the comment @JamesToohey
